packages/widgets-manager/src/utils/index.ts

```diff
@@ -0,0 +1,71 @@
+import { Position, HTMLFloatType } from './../types';
+
+/**
+ * Return an element by its id or throws an error if it can not find one
+ * @param id The dom element id
+ */
+const elementById = (id: string): HTMLElement => {
+  const element = document.getElementById(id);
+  if (element === null) throw Error('Can not find the dom element with id: ' + id);
+  return element;
+}
+
+/**
+ * Helper function to get an element bounding position 
+ * @param elementId The id of the dom element (it must exist)
+ */
+const getElementDomPosition = (elementId: string) => {
+  const element = elementById(elementId);
+  const positionInfo = element.getBoundingClientRect();
+  
+  return positionInfo;
+}
+
+/**
+ * Specialized function to calculate the position of a fixed floating element
+ * @param elementId The id of the dom element
+ */
+export const getElementPositionFixed = (elementId: string): Position => {
+  const domPosition = getElementDomPosition(elementId);
+
+  return {
+    x: domPosition.left,
+    y: domPosition.top
+  };
+}
+
+/**
+ * Specialized function to calculate the position of a default floating element
```

Comment:
all well designed functions are specialized and all pure functions calculate stuff
"Specialized function to calculate" makes no sense

`gets the position of a default floating element` is more than enough